packages/runtime/src/schema.ts

```diff
@@ -818,26 +857,49 @@ class SchemaBuilder {
     name: string,
     model: RuntimeModel,
   ) {
-    this.#mutations[`create${name}`] = mutationWithClientMutationId({
-      name: `Create${name}`,
-      inputFields: () => ({
-        content: { type: new GraphQLNonNull(this.#inputObjects[name]) },
-      }),
-      outputFields: () => ({
-        ...queryFields,
-        document: { type: new GraphQLNonNull(this.#types[name]) },
-      }),
-      mutateAndGetPayload: async (input: { content: Record<string, unknown> }, ctx: Context) => {
-        if (ctx.ceramic.did == null || !ctx.ceramic.did.authenticated) {
-          throw new Error('Ceramic instance is not authenticated')
-        }
-        const document =
-          model.accountRelation.type === 'single'
-            ? await ctx.createSingle(model.id, input.content)
-            : await ctx.createDoc(model.id, input.content)
-        return { document }
-      },
-    })
+    if (model.accountRelation.type === 'single') {
```

Comment:
do we need to special-case single around this whole block like this?  If we can pass through the full `CreateOpts` object, then we can pass that down regardless of if its for a SINGLE or other type of document.  I actually think it would be better to expose the `CreateOpts` for all types of creates.  Ideally I'd love to expose the `UpdateOpts` on mutations as well while we're at it tbh

Reply:
I'd rather avoid creating expectations that these inputs are a direct match for the Ceramic options. ComposeDB is an abstraction over Ceramic that has different needs, notably for updates there is an existing options object.